zjit.c

```diff
@@ -305,6 +305,12 @@ rb_zjit_class_has_default_allocator(VALUE klass)
     return alloc == rb_class_allocate_instance;
 }
 
+VALUE
+rb_zjit_fix_mod_fix(VALUE recv, VALUE obj)
```

Comment:
Can we move `rb_yjit_fix_mod_fix` to jit.c, rename it to `rb_jit_fix_mod_fix`, and share it among YJIT and ZJIT?